src/css/style.css

```diff
@@ -29,6 +29,10 @@
   line-height: 1.75rem;
 }
 
+.section-content-2 {
+  @apply bg-white;
```

Comment:
"Extract repeated combinations of utility classes" is what I found in tailwind's docs explaining why you'd use their apply directive. [Tailwind @apply Docs/Vimeo](https://v1.tailwindcss.com/course/composing-utilities-with-apply)

Why'd you use the @apply here?

Reply:
After reviewing this code. The section-content-2 class should be changed to an inline rather than a class. I have updated the code. Thanks for the question. Great work! 👍